mappings/net/minecraft/world/entity/EntityStore.mapping

```diff
@@ -0,0 +1,17 @@
+CLASS net/minecraft/class_5577 net/minecraft/world/entity/EntityStore
```

Comment:
Would this potentially make sense as `EntityView` or similar to avoid the Impl postfix?

Reply:
`EntityView` is a world interface.

Reply:
Ah okay